zstackctl/zstackctl/utils/portalocker.py

```
@@ -0,0 +1,143 @@
+    LOCK_EX = fcntl.LOCK_EX
+    LOCK_SH = fcntl.LOCK_SH
+    LOCK_NB = fcntl.LOCK_NB
+else:
+    raise RuntimeError("PortaLocker only defined for nt and posix platforms")
+
+if os.name == 'nt':
+    def lock(file, flags):
+        hfile = win32file._get_osfhandle(file.fileno())
+        try:
+            win32file.LockFileEx(hfile, flags, 0, -0x10000, __overlapped)
+        except pywintypes.error, exc_value:
```

Pylint (3.3.7)

[error] Parsing failed: 'multiple exception types must be parenthesized (zstackctl.utils.portalocker, line 94)'

(E0001)